index.d.ts

```diff
@@ -120,6 +120,19 @@ interface ConfigParams {
      */
     redirectUriPath?: string;
 
+    /**
+     * Either a relative path to the application
+     * or a valid URI to an external domain.
+     * The user will be redirected to this after a logout has been performed.
+     */
+    postLogoutRedirectUri?: string;
+
+    /**
+     * If you give a full URI in a `returnTo` parameter that does not start with the `baseURL`,
+     * then you must register it here.
+     */
+    postLogoutRedirectUris?: string[];
```

Comment:
We don't need to confirm or validate the logout URL in the application, the auth server will, necessarily, do that for us. The `postLogoutRedirectUri` option should be the default parameter we send with the logout request. There's nothing wrong with passing along whatever comes through on `returnTo` as the auth server won't redirect back unless it was registered.